src/main/kotlin/com/avail/descriptor/methods/MethodDescriptor.kt

```diff
@@ -382,13 +413,12 @@ class MethodDescriptor private constructor(
 		self.setSlot(SEALED_ARGUMENTS_TYPES_TUPLE, newTuple.makeShared())
 	}
 
-	@AvailMethod
 	override fun o_AddSemanticRestriction(
 		self: AvailObject,
-		restriction: A_SemanticRestriction
+		restrictionSignature: A_SemanticRestriction
```

Comment:
I think `restriction` was a better name.

Reply:
Yeah, it definitely was. It was a bit of laziness on my part — I was renaming it to match the base definitions, so that Kotlin wouldn't complain about named parameters. I can patch it up.